pkg/cloud/openstack/clients/machineservice.go

```diff
@@ -95,6 +100,35 @@ func GetCloudFromSecret(kubeClient kubernetes.Interface, namespace string, secre
 	return clouds.Clouds[cloudName], nil
 }
 
+func authenticatedClient(ao *gophercloud.AuthOptions) (*gophercloud.ProviderClient, error) {
+
+        client, err := openstack.NewClient(ao.IdentityEndpoint)
+        if err != nil {
+                return nil, err
+        }
+        // add HTTPClient includes transport
+	caFile := "/etc//certs/cacert"
+	caCert, err := ioutil.ReadFile(caFile)
+	if err != nil {
+		panic(err)
```

Comment:
no ca file lead to panic?

Reply:
Thanks.
I forgot to remove authenticatedClient function which is not used anymore..